blobfuse/utilities.cpp

```diff
@@ -276,9 +274,6 @@ int azs_getattr(const char *path, struct stat *stbuf)
 
                 stbuf->st_uid = fuse_get_context()->uid;
                 stbuf->st_gid = fuse_get_context()->gid;
-                //stbuf->st_mtime = blob_property.last_modified;
```

Comment:
We need to preserve the correct last_modified time.

Reply:
yes below the comment we are preserving it. This is just the comment.